mods/cnc/maps/gdi10a/map.yaml

```diff
@@ -0,0 +1,898 @@
+MapFormat: 11
+
+RequiresMod: cnc
+
+Title: 10a: Orcastration
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 64,64
+
+Bounds: 6,5,55,57
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: Random
+		Allies: Neutral
+		Enemies: Nod, GDI
+	PlayerReference@neutral:
+		Name: neutral
+		Faction: gdi
+		Allies: neutral
+		Enemies: Nod
+	PlayerReference@Nod:
+		Name: Nod
+		Faction: nod
+		Allies: Nod
+		Enemies: neutral, GDI
+		Color: FE1100
+		Bot: campaign
+	PlayerReference@GDI:
+		Name: GDI
+		Faction: gdi
+		Allies: GDI
+		Enemies: Nod, neutral
+		AllowBots: False
+		Playable: True
+		Required: True
+		LockFaction: True
+		LockColor: True
+		Color: F5D378
+		LockSpawn: True
+		LockTeam: True
+
+Actors:
+	Actor0: sbag
+		Location: 13,10
+		Owner: Neutral
```

Comment:
Should we be changing the original gameplay?  The sandbag wall actually keeps you away one square away from the ridge beside your base where the enemy will march to.  So that space is not good to build on.

Reply:
That's fine then. 👍